src/shared/utils/workspace.ts

```diff
@@ -86,6 +87,10 @@ export function tryProcessBazelFlutterConfig(logger: Logger, config: WritableWor
 		config.localDeviceCommandAdviceMessage = flutterConfig.localDeviceCommandAdviceMessage;
 		config.localMacWarningMessage = flutterConfig.localMacWarningMessage;
 		config.supportsPackageTest = true;
+		// TODO(dantup): Check with helin24 whether this should be enabled. Running
+		//  "dart run test:test --version" is how we check pkg:test capabilities (such
+		//  as whether we can run tests by line number).
+		config.supportsDartRunTest = false;
```

Comment:
@helin24 we added this `supportsPackageTest` flag for Bazel workspaces to tell Dart-Code that it can run tests using `--name` args even though there's no pubspec and we can't verify `package:test` is available.

I've added another flag to check whether it's safe to run `dart run test:test --version` (because we cannot do this in the SDK, for example) when trying to check package:test capabilities.

Can you confirm whether it's valid (and safe) to run internally? If not, we should leave this flag as `false`, but otherwise it can be `true`. Having `true` is required to detect some new features of `package:test` like being able to run tests using their line numbers (`dart test "foo_test.dart?line=10"`) instead of names to avoid some issues with dynamic/duplicated names.

Reply:
(I'm going to land this because I'd like to publish a pre-release version.. this question isn't urgent, as the default is safe - but we can tweak if it if turns out we can do `dart run test:test` here.. if we can't, then version-specific `package:test` functionality like running by line numbers will be disabled, and we can look at another way of doing this if we want to support that)

Reply:
This won't work internally yet - we'd probably need to modify the internal code to parse a line number from similar syntax and/or create a different launch command for internal situations. I'm not sure where this would fall on our priorities to implement; were we thinking of removing the ability to start running tests with text? (I think we had a doc on this and now can't find it..)

Reply:
No plans to remove running tests by name. Depending on how it goes, I might flip the default to using lines in future (to avoid things not working sometimes in ways that aren't obvious to users), but we'd only use it where we know it's supported.

For now I'll just replace the TODO with a comment that this isn't supported, and if at any point it becomes useful to have internally, we can look at changing it then. Thanks!

Reply:
I replaced the TODO with some notes in 32b22cfae095ca49db4088edca8be7f22fc3d0d4. Thanks!